sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkLauncher.java

```diff
@@ -760,66 +763,69 @@ private String logsDir(long now) {
   }
 
   static final DoFn<Event, byte[]> EVENT_TO_BYTEARRAY =
-          new DoFn<Event, byte[]>() {
-            @ProcessElement
-            public void processElement(ProcessContext c) {
-              try {
-                byte[] encodedEvent = CoderUtils.encodeToByteArray(Event.CODER, c.element());
-                c.output(encodedEvent);
-              } catch (CoderException e1) {
-                LOG.error("Error while sending Event {} to Kafka: serialization error",
-                        c.element().toString());
-              }
-            }
-          };
+      new DoFn<Event, byte[]>() {
```

Comment:
Just indentation change? You can remove this change, I modified this as well to remove exception handling in #5247

Reply:
Yes, I'll rollback it